tests/test_vaulty.py

```diff
@@ -44,6 +44,40 @@ def test_list_handles_bad_path(state):
 # Commands
 ##########
 
+def test_cd(state):
+    """cd"""
+    state.pwd = 'foo/narf/'
+    out = cmd_cd(state)
+    assert out is None
+    assert state.pwd == state.home
+
+
+def test_cd__(state):
+    """cd -"""
+    state.pwd = 'foo/narf/'
+    state.oldpwd = 'bar/barf/'
+    out = cmd_cd(state, '-')
+    assert out == 'bar/barf/ is not a valid path'
+    assert state.pwd == 'foo/narf/'
+
+    with patch.object(state, 'list', returns=True):
+        out = cmd_cd(state, '-')
+        assert out is None
+        assert state.pwd == 'bar/barf/'
+
+
+def test_cd_somedir(state):
+    """cd somedir"""
+    out = cmd_cd(state, 'somedir')
+    assert out == 'secret/somedir/ is not a valid path'
+    assert state.pwd == 'secret/'
+
+    with patch.object(state, 'list', returns=True):
```

Comment:
Instead of making a new patch, should use the existing mock logic I made for the state fixture but forgot about